buzzmoon-frontend/buzzmoon-vite/src/Components/Game/Game.jsx

```diff
@@ -1,14 +1,16 @@
 import * as React from 'react';
 import QuestionResult from '../QuestionResult/QuestionResult';
 import QuestionSpeaker from '../QuestionSpeaker/QuestionSpeaker';
-import { useParams } from 'react-router-dom';
+import { Navigate, useNavigate, useParams } from 'react-router-dom';
 import BackendActor from '../BackendActor/backend-actor';
 import './Game.css';
 import GameScore from '../GameScore/GameScore';
 
 export default function Game(props) {
   const {gameID} = useParams();
+  const navigate = useNavigate();
 
+  const [gameData, setGameData] = React.useState({});
   const [questionNumber, setQuestionNumber] = React.useState(0);
```

Comment:
Having **Question Number 0** in the beginning might confuse the user. Is there any way you could just start directly in Question 1 as any other game would?